src/components/Sidebar/Sidebar.component.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { NavLink } from 'react-router-dom';
+import './Sidebar.styles.css';
+import { useAuth } from '../../providers/Auth';
+import { SidebarStyled } from './Sidebar.styled';
+
+const Sidebar = ({ isOpen, toggle }) => {
+  const { authenticated } = useAuth();
+  const domNode = document.getElementById('sidebar-container');
+
+  if (!domNode) {
```

Comment:
What is the purpose of this validation?

Reply:
if we can't find the div for add the content to Portal we don't follow with the render.